phpBB/styles/prosilver/theme/content.css

```diff
@@ -491,18 +491,17 @@ blockquote cite > div {
 
 /* Code block */
 .codebox {
-	padding: 3px;
 	border: 1px solid transparent;
 	font-size: 1em;
 	margin: 1em 0 1.2em 0;
-	overflow-x: scroll;
 	word-wrap: normal;
 }
 
 .codebox p {
 	text-transform: uppercase;
 	border-bottom: 1px solid transparent;
-	margin-bottom: 3px;
+	margin-bottom: 0;
+	padding: 3px 3px 0 3px;
```

Comment:
@Crizz0 Just my opinion, but I think that a website's internal scrollbars should look the same across platforms (see Facebook for example) and that they should be part of the default theme and depend on the developers' taste just like the rest of the website. Of course, the main browser scrollbars should never be touched!

@gooof Certainly I can, but I don't think that it would be a desirable change. I for one wouldn't like it that way. There was a debate for something similar in 3.1, about having the post buttons (e.g. "Report this post" and "Reply with quote") hidden by default and make them appear on hover using a transition, and well, it was preferred to have them visible.